sass/util/_space-misc.scss

```diff
@@ -0,0 +1,3 @@
+// additional, ad-hoc spacing utilities added as needed
+
+.lh-32p { line-height: 32px; }
```

Comment:
it would definitely be better and we should try really hard to only use those but in this case, i think we need something extra, because the left and right side blocks have different font sizes and thus getting them to align requires equating the line heights (and the 4 basscss line height helpers [`line-height-*`] are relative values not absolute heights, e.g., 1.25 or 1.5

though maybe i'm missing a better way, in which case i'm all ears!